package.json

```diff
@@ -9,10 +9,28 @@
     "dev-start": "ts-node ./src/index.ts",
     "dev": "nodemon -e ts -w ./src -x npm run dev-start",
     "build-dev": "tsc && npm start",
-    "build": "rm -rf dist && tsc cp -r ./src/templates ./dist/templates",
-    "lint": "tsc --noEmit && eslint \"src/*/**/*.{js,ts,tsx}\" --quiet --fix",
+    "build": "rm -rf dist && tsc",
+    "lint": "tsc --noEmit && eslint 'src/*/**/*.{js,ts,tsx}' --quiet --fix",
     "migrate": "ts-jest config:migrate jest.config.js"
   },
+  "jest": {
```

Comment:
What about putting these configurations`jest.config.js`?

Reply:
Yeah, definitely , I will do it...